lib/yoti/protobuf/v1/attribute_public_api/attribute.pb.rb

```diff
@@ -13,6 +13,7 @@ class ContentType < ::Protobuf::Enum
           define :JPEG, 2
           define :DATE, 3
           define :PNG, 4
+          define :BYTES, 5
```

Comment:
is this auto generated or you wrote it yourself? If it's the latter, why bytes and not json?

Reply:
Yes I wrote it myself, I put bytes in reference to Node but will change it to JSON.